src/components/WorkflowForms/NewClientSignup/IntroAndInitialAdvice/IntroAndInitialAdvice.jsx

```
@@ -5,21 +5,33 @@ import mva_utils from '../../../../utilities/functions.jsx';
   const [isLoading, setIsLoading] = useState(false);
 
   const handleNext = async () => {
     setIsLoading(true);
     try {
-      await api.complete_workflow_step({
+      const response = await api.complete_workflow_step({
         workflow_business_key: caseId,
         task_definition_key: "introAndInitialAdvice"
       });
+      
+      if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)